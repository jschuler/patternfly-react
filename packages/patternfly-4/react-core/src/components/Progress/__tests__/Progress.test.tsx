// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { Progress } from '../Progress';

describe('Progress test', () => {
  it('Progress should match snapshot', () => {
    const view = mount(
      <Progress
        className={"''"}
				size={null}
				measureLocation={'outside'}
				variant={'danger'}
				title={"''"}
				label={null}
				value={0}
				id={"''"}
				min={0}
				max={100}
				valueText={"null"}
      />);
    expect(view).toMatchSnapshot();
  });
});
