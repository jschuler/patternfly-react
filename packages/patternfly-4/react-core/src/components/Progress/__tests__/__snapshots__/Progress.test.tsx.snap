// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress test Progress should match snapshot 1`] = `
<Progress
  className="''"
  id="''"
  label={null}
  max={100}
  measureLocation="outside"
  min={0}
  size={null}
  title="''"
  value={0}
  valueText="null"
  variant="danger"
>
  <div
    aria-valuetext="null"
    className="pf-c-progress pf-m-danger pf-m-outside ''"
    id="''"
    role="progressbar"
  >
    <Component
      ariaProps={
        Object {
          "aria-describedby": "''-description",
          "aria-valuemax": 100,
          "aria-valuemin": 0,
          "aria-valuenow": 0,
          "aria-valuetext": "null",
        }
      }
      label={null}
      measureLocation="outside"
      parentId="''"
      title="''"
      value={0}
      variant="danger"
    >
      <div
        className="pf-c-progress__description"
        id="''-description"
      >
        ''
      </div>
      <div
        className="pf-c-progress__status"
      >
        <span
          className="pf-c-progress__measure"
        >
          0%
        </span>
        <span
          className="pf-c-progress__status-icon"
        >
          <TimesCircleIcon
            color="currentColor"
            noVerticalAlign={false}
            size="sm"
            title={null}
          >
            <svg
              aria-hidden={true}
              aria-labelledby={null}
              fill="currentColor"
              height="1em"
              role="img"
              style={
                Object {
                  "verticalAlign": "-0.125em",
                }
              }
              viewBox="0 0 512 512"
              width="1em"
            >
              <path
                d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"
                transform=""
              />
            </svg>
          </TimesCircleIcon>
        </span>
      </div>
      <Component
        ariaProps={
          Object {
            "aria-describedby": "''-description",
            "aria-valuemax": 100,
            "aria-valuemin": 0,
            "aria-valuenow": 0,
            "aria-valuetext": "null",
          }
        }
        value={0}
      >
        <div
          aria-describedby="''-description"
          aria-valuemax={100}
          aria-valuemin={0}
          aria-valuenow={0}
          aria-valuetext="null"
          className="pf-c-progress__bar"
        >
          <div
            className="pf-c-progress__indicator"
            style={
              Object {
                "width": "0%",
              }
            }
          >
            <span
              className="pf-c-progress__measure"
            />
          </div>
        </div>
      </Component>
    </Component>
  </div>
</Progress>
`;
